src/eventProcessor/GoogleAnalyticsEventProcessor.js

```diff
@@ -0,0 +1,129 @@
+goog.module('measurementLibrary.eventProcessor.GoogleAnalyticsEventProcessor');
+
+/**
+ * A class that processes events pushed to the data layer
+ * by constructing and sending Google Analytics events via
+ * Measurement Protocol.
```

Comment:
Would add "Measurement Protocol (App + Web)" to disambiguate that it's not using the version of measurement protocol used by analytics.js.